File: Java/shuffle_the_array/shuffle_the_array.java
package shuffle_the_array;

class shuffle_the_array{
    public int[] shuffle(int[] nums, int n) {
        int[] result = new int[nums.length];
        int temp = 0;

        for(int i =0; i<nums.length; i++){
            if(i % 2 == 0){
                result[i] = nums[temp];
                temp++;
            } else {
                result[i] = nums[n];
                n++;
            }
        }

        return result;
    }
}

File: Python/shuffle_the_array/shuffle_the_array.py
class Solution(object):
    def shuffle(self, nums, n):
        """
        :type nums: List[int]
        :type n: int
        :rtype: List[int]
        """
        result = []
        count = n
        temp = 0

        for x in range(len(nums)):
            if x % 2 == 0:
                result.append(nums[temp])
                temp = temp + 1
            else:
                result.append(nums[count])
                count = count + 1

        return result